solutions/product_solutions/GVAMPP/GVAMPP_Manager_Using.md

```diff
@@ -19,3 +21,21 @@ The sidebar on the left of the Grass Valley AMPP Manager app contains buttons th
 | ![Snapshots Control](~/dataminer/images/GVAMPP_Button2_Icon.png) | Opens the *Snapshots Control* page. Similar to *Workloads Control*, this displays a table of all discovered snapshots, allowing users to start or stop selected ones. |
 | ![Configuration](~/dataminer/images/GVAMPP_Button3_Icon.png) | Opens the *Setup* page, where users can [configure the external component](xref:GVAMPP_Manager_Installing#installing-and-configuring-the-communication-component) and create a GV AMPP Manager element. |
 | ![About](~/dataminer/images/GVAMPP_Button4_Icon.png) | Opens the *About* page, which shows the application version information. |
+
+## Using the Workload Application Element Configurator
+
+On the Workloads Control page, the **Workload Element** column displays the currently assigned application element. An application element utilizes a specialized connector that processes the configuration and state messages specific to a particular application type. This connector also includes configuration parameters that allow changes of the workload. You can for example set the recording file or start a clip player.
```

Comment:
As a user, I think I'd still be left with some questions after reading this: how do I set the recording file and start a clip player? This isn't really explained below.